docs/docsite/rst/getting_started/get_started_inventory.rst

```diff
@@ -38,6 +38,8 @@ Continue getting started with Ansible and build an inventory as follows:
    .. note::
       Pass the ``-u`` option with the ``ansible`` command if the username is different on the control node and the managed node(s).
 
+      Pass the ``-k`` option if SSH authentication on the managed node(s) requires a password.
+
```

Comment:
I may be misremembering, but I thought using -k to pass in a password isn't actually the preferred approach because it weakens security.  If you feel it's important in the getting started to show how to deal with a managed host that requires a password, please reword this to say you can use -k for this example but should implement ansible vault for any production system.

Reply:
i've removed this as now i noticed that the author mentioned putting keys on managed nodes. thanks